qutebrowser/config/configdata.yml

```diff
@@ -3194,3 +3194,26 @@ bindings.commands:
 
     * register: Entered when qutebrowser is waiting for a register name/key for
       commands like `:set-mark`.
+
+## logging
+
+logging.level.ram:
+  default: null
+  type:
+    name: String
+    none_ok: true
+    # levels match those in qutebrowser/utils/log.py
+    valid_values: ['VDEBUG', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
```

Comment:
Maybe write a custom small config type for this? Could be similar to `configtypes.NewTabPosition` (i.e. quite straightforward), and ge tthe levels directly from `utils.log` so that this doesn't need to be duplicated (twice) in `configdata.yml`.